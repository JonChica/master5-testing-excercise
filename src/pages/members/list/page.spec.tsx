import * as React from 'react';
import { shallow } from 'enzyme';
import { MemberListPage } from './page';

describe('pages/members/list/page specs', () => {
  it('should render as expected when passing required properties', () => {
    // Arrange
    const props = {
      members: [
        {
          id: 1,
          name: 'test name 1',
          avatarUrl: 'test avatarUrl 1',
        },
        {
          id: 2,
          name: 'test name 2',
          avatarUrl: 'test avatarUrl 2',
        },
      ],
    };

    // Act
    const component = shallow(<MemberListPage {...props} />);

    // Assert
    expect(component).toMatchSnapshot();
  });
});
