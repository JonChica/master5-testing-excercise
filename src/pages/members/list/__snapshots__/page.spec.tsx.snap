// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/page specs should render as expected when passing required properties 1`] = `
<Fragment>
  <h2>
    Members
  </h2>
  <Component
    members={
      Array [
        Object {
          "avatarUrl": "test avatarUrl 1",
          "id": 1,
          "name": "test name 1",
        },
        Object {
          "avatarUrl": "test avatarUrl 2",
          "id": 2,
          "name": "test name 2",
        },
      ]
    }
  />
</Fragment>
`;
